sql-files/main.sql

```diff
@@ -916,3 +918,53 @@ CREATE TABLE IF NOT EXISTS `storage` (
   PRIMARY KEY (`id`),
   KEY `account_id` (`account_id`)
 ) ENGINE=MyISAM;
+
+CREATE TABLE IF NOT EXISTS `rodex_items` (
+	`id` INT(11) NOT NULL AUTO_INCREMENT,
+	`mail_id` BIGINT(20) NOT NULL DEFAULT '0',
+	`nameid` INT(11) NOT NULL DEFAULT '0',
+	`amount` INT(11) NOT NULL DEFAULT '0',
+	`equip` INT(11) UNSIGNED NOT NULL DEFAULT '0',
+	`identify` SMALLINT(6) NOT NULL DEFAULT '0',
+	`refine` TINYINT(3) UNSIGNED NOT NULL DEFAULT '0',
+	`attribute` TINYINT(4) NOT NULL DEFAULT '0',
+	`card0` SMALLINT(11) NOT NULL DEFAULT '0',
+	`card1` SMALLINT(11) NOT NULL DEFAULT '0',
+	`card2` SMALLINT(11) NOT NULL DEFAULT '0',
+	`card3` SMALLINT(11) NOT NULL DEFAULT '0',
+  `opt_idx0` SMALLINT(5) UNSIGNED NOT NULL DEFAULT '0',
+  `opt_val0` SMALLINT(5) NOT NULL DEFAULT '0',
+  `opt_idx1` SMALLINT(5) UNSIGNED NOT NULL DEFAULT '0',
+  `opt_val1` SMALLINT(5) NOT NULL DEFAULT '0',
+  `opt_idx2` SMALLINT(5) UNSIGNED NOT NULL DEFAULT '0',
+  `opt_val2` SMALLINT(5) NOT NULL DEFAULT '0',
+  `opt_idx3` SMALLINT(5) UNSIGNED NOT NULL DEFAULT '0',
+  `opt_val3` SMALLINT(5) NOT NULL DEFAULT '0',
+  `opt_idx4` SMALLINT(5) UNSIGNED NOT NULL DEFAULT '0',
+  `opt_val4` SMALLINT(5) NOT NULL DEFAULT '0',
+	`expire_time` INT(11) UNSIGNED NOT NULL DEFAULT '0',
+	`bound` TINYINT(1) UNSIGNED NOT NULL DEFAULT '0',
+	`unique_id` BIGINT(20) UNSIGNED NOT NULL DEFAULT '0',
+	PRIMARY KEY (`id`),
+  KEY `mail_id` (`mail_id`)
+) ENGINE=InnoDB;
+
+CREATE TABLE IF NOT EXISTS `rodex_mail` (
+	`id` BIGINT(20) NOT NULL AUTO_INCREMENT,
```

Comment:
Can you rename this to `mail_id`, for consistency with the other table (we usually tend to re-use the column names to avoid confusion)